react/SearchWrapper.tsx

```diff
@@ -73,34 +74,53 @@ const getPageEventName = (
   return (mapEvent[category] || fallbackView) as PageEventName
 }
 
+const getDecodeURIComponent = (encodedURIComponent: string) => {
+  try {
+    return decodeURIComponent(encodedURIComponent)
+  } catch {
+    return encodedURIComponent
+  }
+}
+
 interface GetTitleTagParams {
   titleTag: string
   storeTitle: string
   term?: string
   pageTitle?: string
+  pageNumber?: number
+  removeStoreNameTitle?: boolean
 }
 
 const getTitleTag = ({
   titleTag,
   storeTitle,
   term,
   pageTitle,
+  pageNumber = 0,
+  removeStoreNameTitle,
 }: GetTitleTagParams) => {
+  const titleNumber = pageNumber > 0 ? ` #${pageNumber}` : ''
```

Comment:
This should be:

```suggestion
  const titleNumber = pageNumber > 0 ? ` #${pageNumber + 1}` : ''
```

right? Then you wouldn't show `#1`.

Reply:
in the first page don't show page number.
**pageNumber** depends of the value of **page**_(url query params)_, **page param** it set from page 2 onwards.

Reply:
Yeah, you're right.
![image](https://user-images.githubusercontent.com/15680320/153222148-8b3a8968-ef15-415d-9a70-ad20f1deff5d.png)
I will resolve this conversation, @victorhmp .